src/views/htmlcontent/test/app.component.spec.ts

```diff
@@ -270,14 +277,101 @@ describe('AppComponent', function (): void {
             ele = fixture.nativeElement;
         });
 
+        it('should have started showing messages after the batch start command', () => {
+            let dataService = <MockDataService> fixture.componentRef.injector.get(DataService);
+            dataService.sendWSEvent(batchStart);
+            fixture.detectChanges();
+
+            let results = ele.querySelector('#results');
+            expect(results).toBeNull('results pane is visible');
+
+            // Messages should be visible
+            let messages = ele.querySelector('#messages');
+            expect(messages).not.toBeNull('messages pane is not visible');
+            expect(messages.className.indexOf('hidden')).toEqual(-1);
```

Comment:
I didn't learn this till later, but this probably isn't the best way to determine if something is visible. It can have the class, but the class can be overwritten by a different class. A better way would be to get the [computed style](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle) of the element and check the display property.